tools/fuchsia/build_fuchsia_artifacts.py

```diff
@@ -196,8 +196,7 @@ def BuildTarget(runtime_mode, arch, product):
   out_dir = 'fuchsia_%s_%s' % (runtime_mode, arch)
   flags = [
       '--fuchsia',
-      # The source does not require LTO and LTO is not wired up for targets.
-      '--no-lto',
+      '--lto',
```

Comment:
This is the default. Let's get rid of the flag altogether. This stuff gets copy pasted a lot and it would be unfortunate if an unopt variant was built with lto.